src/main/java/entities/InvitationInterface.java

```diff
@@ -0,0 +1,22 @@
+package entities;
+
+import entities.Task;
+import entities.User;
+
+public interface InvitationInterface {
```

Comment:
Is it necessary to implement an interface for an entity? 
Case 1: the entity is going to be only called by other entities and UseCase classes. Then an interface is not needed since UseCase can call entities without breaking clean architecture.
Case 2: the entity is going to be passed around in layers other than UseCase and entities. Then since entities are not supposed to be passed into these other layers, the interface is not necessary and a ResponseModel or similar data object should be used.